switchdev_utils/config-service/uninstall.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
some basic indication to the user using echo to tell the user we are uninstalling and that uninstall it done would be helpful